### PR TITLE
device.mk: build va and vaapi into Android

### PR DESCRIPTION
**Pull request by LindaYu17**
Jira: https://01.org/jira/browser/AIA-65
Test: vainfo

Signed-off-by: Linda Yu <linda.yu@intel.com>

### CODE REVIEW

**Comment by xuguangxin** [comment]
@kalyankondapally , please help to view. I can merge it after you +1 for this.

**Comment by kalyankondapally** [comment]
Ya testing all changes together before we can merge all this

File: androidia_64/device.mk [hunk 1]
```diff
@@ -167,6 +167,14 @@ PRODUCT_PACKAGES += \
     vibrator.default.so \
     keystore.default.so
 
+# libva, vaapi
```
Comment (kalyankondapally) [inline androidia_64/device.mk]: You need to add this to mixin file here: https://github.com/android-ia/device-androidia-mixins/blob/master/groups/media/android_ia/product.mk

Comment (LindaYu17) [inline androidia_64/device.mk]: https://github.com/android-ia/device-androidia-mixins/pull/28

like this?

**Comment by kalyankondapally** [comment]
yes, merged the mixin update

**Pull request closed without merging**